docs/model_server_rest_api_completions.md

```diff
@@ -73,7 +73,7 @@ curl http://localhost/v3/completions \
 #### Multinomial sampling specific
 | Param | OpenVINO Model Server | OpenAI /completions API | vLLM Serving Sampling Params | Type | Description |
 |-------|----------|----------|----------|---------|-----|
-| temperature | ✅ | ✅ | ✅ | float (default: `0.0`) | The value is used to modulate token probabilities for multinomial sampling. It enables multinomial sampling when set to `> 0.0`. |
+| temperature | ✅ | ✅ | ✅ | float (default: `1.0`) | The value is used to modulate token probabilities for multinomial sampling. It enables multinomial sampling when set to `> 0.0`. |
```

Comment:
Please update also this line:
https://github.com/openvinotoolkit/model_server/blob/main/src/llm/apis/openai_completions.cpp#L229

And other parts of the code where we refer to deafult=0.0 which is outdated now